Kiosk/Bid Fulfillment/ConfirmYourBidArtsyLoginViewController.swift

```diff
@@ -83,14 +99,19 @@ class ConfirmYourBidArtsyLoginViewController: UIViewController {
         }
     }
 
+    override func viewWillDisappear(animated: Bool) {
+        super.viewWillDisappear(animated)
+        _viewWillDisappear.onNext()
```

Comment:
Yeah, I thought about abstracting it a bit, but I'm going to wait on https://github.com/ReactiveX/RxSwift/issues/181 to see if someone else fixes it for me :wink: